controllers/util/mw_util.go

```diff
@@ -218,6 +220,40 @@ func Namespace(name string) *corev1.Namespace {
 	}
 }
 
+func (mwu *MWUtil) GetManifest(
+	manifests []ocmworkv1.Manifest,
+	gvk schema.GroupVersionKind,
+	name, namespace string) (*runtime.RawExtension, error) {
```

Comment:
(nit) I would return a `Manifest` object ptr when calling `GetManifest` rather than RawExtension ptr.

Reply:
Ah, yes. I would rename the function to `GetRawExtention`, the intent being to retain all manifest object manupulation within mwu_util and not have other files go from Manifest to Object.

Does that sound ok?

Reply:
Great, that works.

Reply:
Done!